src/pxUtil.cpp

```diff
@@ -1023,13 +1023,22 @@ rtError pxLoadSVGImage(const char* buf, size_t buflen, pxOffscreen& o, int  w /*
     return RT_FAIL;
   }
 
-  NSVGimage *image = nsvgParse( (char *) buf, "px", 96.0f); // 96 dpi (suggested default)
+  // NOTE:  'nanosvg' is *destructive* to the SVG source buffer
+  //
+  //        Pass it a copy !
+  //
+  void *buf_copy = malloc(buflen);
+  memcpy(buf_copy, buf, buflen);
+  
+  NSVGimage *image = nsvgParse( (char *) buf_copy, "px", 96.0f); // 96 dpi (suggested default)
   if (image == NULL)
   {
     rtLogError("SVG:  Could not init decode SVG.\n");
     return RT_FAIL;
```

Comment:
At fist glance it looks like there is a missing `free(buf_copy);` just before `return RT_FAIL;`